README.md

```diff
@@ -70,7 +70,7 @@ Commands:
 
 Examples:
   npx fpl install hl7.fhir.r5.core@current
-  fpl install hl7.fhir.r4.core@4.0.1 hl7.fhir.us.core@4.0.0 --save ./myProject
+  fpl install hl7.fhir.r4.core@4.0.1 hl7.fhir.us.core@4.0.0 --cachePath ./myProject
```

Comment:
Downloading FHIR core is kind of an edge case (especially `current`, because IG Publisher doesn't support building w/ R5 current).  Can you just use US Core and mCODE as examples instead?  If you want to show `current`, you could use US Core as the example.

Reply:
Both should be fixed in [2882f24](https://github.com/standardhealth/fhir-package-loader/pull/1/commits/2882f24f7a365ab9cc8cfdab03edc45005e9be1a). Thanks for catching that! It's just static text that gets added to the `--help` text so I missed it! And good to know about FHIR core. It's still used in a bunch of tests and there's logic for downloading it in the code, but I think that should all be fine since we probably want it to work and just want to advertise a different example.